app/src/js/components/Executions/execution-status.js

```diff
@@ -165,6 +169,11 @@ class ExecutionStatus extends React.Component {
           <dt>Logs:</dt>
           <dd><Link to={'/executions/execution/' + executionStatus.execution.name + '/logs'} title={executionStatus.execution.name + '/logs'}>View Execution Logs</Link></dd>
           <br />
+          { kibanaLink && kibanaLink.length ? (<div>
```

Comment:
Can you get a verification from @jjmccoy that we want to show both links.  I think it's fine this way, but want a second opinion.

Reply:
I don't have an issue with redundancy but if you are going to 'View Execution Logs' and that takes you to executions/execution (little confused on where we are going from the 'Execution Details' if we are not going to 'Kibana' at this link) once and click on a log there doesn't that redirect you to Kibana? I think the user is going to wonder what's the difference between 'Logs' and 'Kibana Logs'.

Reply:
so executions/execution is the execution detail for the given execution. (nice sentence).
Looks like this:

![Screen Shot 2020-02-19 at 7 46 27 AM](https://user-images.githubusercontent.com/479480/74845253-f5a14300-52eb-11ea-8e48-a9fd41982d89.png)

Reply:
It's just a view of the same execution logs inside of the dashboard.  I think it's harder to understand, but operators are going to have to get used to that view if we're giving them kibana.
![Screen Shot 2020-02-19 at 7 48 25 AM](https://user-images.githubusercontent.com/479480/74845473-4fa20880-52ec-11ea-8511-4e330bfc9c59.png)

Reply:
After chewing it over I think one link is better (kibana if it's set up, dashboard page if it's not). Would they even use the regular dashboard page if Kibana is available?

Reply:
Okay cool. So I'm assuming that the 'Execution Logs' is more of a highlight with brief details for quick overview and the 'Kibana Logs' will be for more discovery?

Reply:
@jjmccoy Yeah, I would think the link to kibana will take you to the records for the execution, but then you're right there ready to do discovery.

@bt6453 If kibana isn't available, what happens to that executions/execution link?  is there any data to show?

Reply:
I tested for myself, it makes the request to the cumlusAPI/logs  endpoint and gets the same data back. Noice.